azure-pipelines.yml

```diff
@@ -388,9 +388,22 @@ stages:
       - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -restore -binaryLogName Restore.binlog 
         displayName: Restore
 
-      - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -build -pack -publish -sign -binaryLogName Build.binlog
+      - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -build -pack -publish -sign -binaryLogName Build.binlog /p:DotnetPublishUsingPipelines=true
         displayName: Build
 
+      - task: PowerShell@2
+        displayName: Publish Symbols Dry-Run
+        inputs:
+          filePath: eng\common\sdk-task.ps1
+          arguments: -task PublishToSymbolServers /p:DryRun="true" -restore -msbuildEngine dotnet
```

Comment:
Borrowed this bit from [Arcades PR CI](https://github.com/dotnet/arcade/blob/ed14da5934ffb536cff8f41f8b5719334524cbed/azure-pipelines-pr.yml#L163). Since we are using v3 publishing, we need to emulate what the Maestro Build Promotion pipeline does to publish symbols.

Reply:
Ah nice - this means we can have some verification of publish symbols in regular CI now right?

Reply:
Well, it was already happening in this job, but was using v2 publishing. That caused CI to fail now that we need the exclusions file. This symbol publishing method supports the exclusion file.